docs/recipes/close-account.md

```diff
@@ -0,0 +1,30 @@
+# Close Account
+
+In accounting, a _closing entry_ zeroes a (temporary) account's balance.
```

Comment:
I know that some of the Investopedia articles etc. reference a temporary account. But this was confusing for me, since I've never seen closing entries described in this way before (no account is really temporary, because you keep them around forever). It's more that some accounts are for workings. Stepping back, we can remove "temporary" here completely since it's not the main point, and rather describe this as:

> In accounting, a closing entry calculates the net debit or credit balance for an account and then credits or debits this balance respectively, to zero the account's balance and move the balance to another account.

Reply:
Also, what often happens, is you close the balance at the year end, and then you might use the account again into the next year. So the account sticks around.